### PR TITLE
fix/rr479-changing-fonts-to-fira-sans

### PR DESCRIPTION
**Pull request by JacOng17**
# What has changed and why?
Added Fira/Fira-Bold fonts & files needed.

# How to test this change
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: app/assets/stylesheets/admin/rebate_form.css [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/* Fonts
+=============================== */
+@font-face {
```
Comment (CaraHill) [inline app/assets/stylesheets/admin/rebate_form.css]: These should go in a general file like `application.css` as opposed to a specific file like this one, as fonts are used globally throughout the app.

Comment (JacOng17) [reply app/assets/stylesheets/admin/rebate_form.css]: :+1:  Will shift it

File: app/assets/stylesheets/pdf.css [hunk 2]
```diff
@@ -11,7 +11,7 @@ body {
 
 * {
   font-family: "Fira Sans", sans-serif;
-  color: rgb(51, 51, 51);
+  color: #333333;
```
Comment (CaraHill) [inline app/assets/stylesheets/pdf.css]: You can roll back all of the changes you've made to the pdf as I've updated it all pretty significantly in my branch.

Comment (JacOng17) [reply app/assets/stylesheets/pdf.css]: :+1:

File: app/assets/stylesheets/pdf.css [hunk 3]
```diff
@@ -65,7 +65,7 @@ body {
 .field,
 .declaration {
   margin-bottom: 10px;
-  color: rgb(51, 51, 51);
+  color: #333333;
```
Comment (CaraHill) [inline app/assets/stylesheets/pdf.css]: Oh sorry, I did comment on both, but I meant to rollback all of your changes to pdf. 🙂

Comment (JacOng17) [reply app/assets/stylesheets/pdf.css]: Oops, sorry. Fixing it now.

**Pull request merged**